protocol/x/clob/keeper/orders.go

```diff
@@ -241,6 +244,97 @@ func (k Keeper) PlaceShortTermOrder(
 	return orderSizeOptimisticallyFilledFromMatchingQuantums, orderStatus, err
 }
 
+// DeliverMsgCancelOrder delivers a MsgCancelOrder.
+func (k Keeper) DeliverMsgCancelOrder(
```

Comment:
nit: should we just leave this in `msg_server_cancel_order.go`? `orders.go` is kind of huge already